src/resources/CaseAssist/CaseAssist.ts

```diff
@@ -0,0 +1,28 @@
+import API from '../../APICore';
+import {New, PageModel} from '../BaseInterfaces';
+import Resource from '../Resource';
+import {CaseAssistListOptions, CaseAssistModel} from './CaseAssistInterfaces';
+
+export default class CaseAssist extends Resource {
```

Comment:
Do you think the naming should reflect the fact that those routes are use for configuration ?

Reply:
As in calling things "CaseAssistConfiguration" or "CaseAssistConfig" ? If so then yes because as I'm doing the front end action functions I'm seeing that makes more sense

Reply:
That was what I have in mind

Reply:
Nice! Well that's what i did. 👍